packages/peregrine/lib/talons/WishlistPage/useWishlistItem.js

```diff
@@ -12,6 +12,16 @@ const dialogs = {
     MORE_ACTIONS: 3
 };
 
+const SUPPORTED_PRODUCT_TYPES = ['SimpleProduct', 'ConfigurableProduct'];
+
+const mergeSupportedProductTypes = (supportedProductTypes = []) => {
+    if (supportedProductTypes.length >= 0) {
```

Comment:
This might fail if `supportedProductTypes` is `null`. `default param` only will pick if its `undefined`.

![image](https://user-images.githubusercontent.com/35203638/123444993-1727d400-d59d-11eb-8229-fe38a3fccd70.png)

Instead try something like this:

```es6
const mergeSupportedProductTypes = (supportedProductTypes = []) => {
    const newSupportedProductTypes = [...SUPPORTED_PRODUCT_TYPES]

    if (supportedProductTypes) {
       newSupportedProductTypes.push(...supportedProductTypes)
    }

    return newSupportedProductTypes;
};
```

![image](https://user-images.githubusercontent.com/35203638/123445806-e005f280-d59d-11eb-9304-0dd1f97f6c23.png)

Reply:
memo can bring here 1-3 ms i did short benchmark but i dobt know if it wohrt it to memo it https://codesandbox.io/s/affectionate-shamir-4mm8n?file=/src/App.js